### PR TITLE
Revert to original datapacket structure

### PR DESCRIPTION
**Pull request by kevinl120**

### CODE REVIEW

File: Software/main/main.cpp [hunk 1]
```diff
@@ -30,22 +30,21 @@ int main() {
 
     // TODO: flight loop
     for (int count  = 0; count < 100; count++) {
-        for (int i = 0; i < sensors.size(); i++) {
-            // Polls data from sensor #i
+        for (int i = 0; i < sensors.size(); ++i) {
```
Comment (ghuser1023) [inline Software/main/main.cpp]: why?

File: Software/spartan/src/data/mdp.h [hunk 2]
```diff
@@ -100,14 +110,14 @@ namespace spartan {
...
     private:
         float m_payload_voltage;
         float m_payload_current;
     };
 
     class PayloadDataPacket {};
-     */
 } // namespace spartan
 
 #endif // MDP_H_INCLUDED
```
Comment (ghuser1023) [inline Software/spartan/src/data/mdp.h:38]: Would prefer these (non-MDP) declarations get moved to `datapacket.h` and the implementations get moved to `datapacket.cpp`; cleaner structure that way.

**Review by ghuser1023 — COMMENTED** [review]
looks good, some comments

File: Software/main/main.cpp (hunk 1 above)
Comment (kevinl120) [reply Software/main/main.cpp]: No reason--spaces vs tabs. Changed to i++ for loop increment consistency.

File: Software/spartan/src/data/mdp.h (hunk 2 above)
Comment (kevinl120) [reply Software/spartan/src/data/mdp.h:38]: Makes sense--done.

**Review by ghuser1023 — APPROVED** [review]
Approved; merging.

**Pull request merged**